core/Plugin.php

```diff
@@ -451,6 +451,19 @@ public static function getPluginNameFromNamespace($namespaceOrClassName)
     }
 
     /**
+     * Override this method in your plugin class if you want your plugin to be loaded during tracking.
+     *
+     * Note: If you define your own dimension or handle a tracker event, your plugin will automatically
+     * be detected as a tracker plugin.
+     *
+     * @return bool
+     */
+    public function isTrackerPlugin()
```

Comment:
So there will be both approaches possible? I'd personally try to avoid it for mentioned reasons unless they maybe want to replace an existing request processor (which would be ok that way).

Just to be sure: If a plugin is deactivated, the `config.php` of such a deactivated plugin is not loaded right?